src/main/java/ApplyToGroup/ApplyToGroupPresenter.java

```diff
@@ -0,0 +1,9 @@
+package ApplyToGroup;
+
+public interface ApplyToGroupPresenter {
+
+    ApplyToGroupResponseModel prepareSuccessView(ApplyToGroupResponseModel user);
```

Comment:
I think you should rename the parameter to be more descriptive, maybe add a doc string. The response model is carrying info about the user and group, right?